compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -61,31 +61,38 @@ void compute_sym_scale_zp(float min, float max, float &scaling_factor, int64_t &
 void compute_asym_scale_zp(float min, float max, float &scaling_factor, int64_t &zp,
                            float &nudged_min, float &nudged_max)
 {
-  assert(min != max);
```

Comment:
Can you add an assertion if you don't handle the case min <= max.

`assert(min <= max);`